services/rclone-http/default.nix

```diff
@@ -0,0 +1,64 @@
+# SPDX-FileCopyrightText: 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  pkgs,
+  lib,
+  config,
+  ...
+}:
+with lib; let
+  cfg = config.services.rclone-http;
+in {
+  options.services.rclone-http = {
+    enable = mkEnableOption "rclone-http service";
+
+    listenAddress = mkOption {
+      type = types.str;
+      default = "localhost:8080";
+      description = "Which IP address and port the server should listen on";
```

Comment:
unix:// is also supported, and we call it with that scheme in the binary cache config. Please extend that description to mention unix sockets are also supported. Maybe the rclone --help output has a concise description we can use here.

Reply:
This was from the rclone docs. The --help output also has no mention of unix:// weirdly enough